.env.development.local.sample

```diff
@@ -1,18 +0,0 @@
-NEXT_PUBLIC_GOOGLE_API_KEY="<Google API key>"
-
-NEXT_PUBLIC_GOOGLE_TAG_MANAGER_AUTH="<GTM auth string>"
-NEXT_PUBLIC_GOOGLE_TAG_MANAGER_ENV="<GTM env string>"
-NEXT_PUBLIC_GOOGLE_TAG_MANAGER_ID="<GTM id>"
-
-NEXT_PUBLIC_ISS_BASE_URL="<ISS 3 base url>"
-NEXT_PUBLIC_ISS_VERSION="3"
-NEXT_PUBLIC_ISS_API_KEY="<ISS 3 API key>"
-
-NEXT_PUBLIC_STRAPI_URL="<Base url of the Strapi-based Ask Izzy CMS>"
-
-NEXT_PUBLIC_PROXY_DOMAIN_SUFFIX="external.askizzy.localhost"
-DOMAINS_TO_PROXY="cloud.typography.com, www.google-analytics.com, www.googletagmanager.com, maps.googleapis.com, csi.gstatic.com, fonts.gstatic.com, tagmanager.google.com, fonts.google.com, www.google-analytics.com, google-analytics.com, stats.g.doubleclick.net, www.gstatic.com, recaptcha.net, widget.usersnap.com, resources.usersnap.com"
-NEXT_PUBLIC_DOMAINS_ALLOWED_WITHOUT_PROXY="iss3.dev.ixapps.org, ask-izzy-cms.dev.ixapps.org"
-
-RECAPTCHA_SECRET_KEY=
-NEXT_PUBLIC_RECAPTCHA_SITE_KEY=
```

Comment:
We want to keep this file in the repo for future people who clone the repo to use